canonical-web-and-design/vanilla-framework
# configurable table layout

Update vanilla dep to v2.0.1

## Done
1) Makes table-layout configurable via a variable ($table-layout)
2) Provides utility to switch to the alternative, i.e. :
- if $table-layout is set to 'auto', adds a utility class called .u-table-layout--fixed
- if $table-layout is set to 'fixed' (which is the default), adds a utility class called .u-table-layout--auto



## QA

- Pull code
- Run `./run serve --watch`
- Open /examples/base/table/
- add class 'u-table-layout--auto', verify it applies an override
- change $table-layout to 'auto' in a repo that imports vanilla;
- copy the table example to your repo, and add class 'u-table-layout--fixed' to the table
- verify the utility overrides as expected

## Details

Fixes #2374
